test/provision/k8s_install.sh

```diff
@@ -257,8 +257,7 @@ sudo systemctl restart $MOUNT_SYSTEMD
 sudo rm -rfv /var/lib/kubelet
 
 # Allow iptables forwarding so kube-dns can function.
-sudo iptables -P FORWARD ACCEPT
-sudo ip6tables -P FORWARD ACCEPT
+sudo iptables --policy FORWARD ACCEPT
```

Comment:
Is this line still necessary?

Reply:
Hmm, that one was originally added back in 1d16b21271fc7e258fd6c6b1e3c5e2fc4f5ba24f. So I'm not 100% sure whether we still need it or not (/cc @ianvernon ) wrt kube-dns. Given this is only for the test suite, I'll remove it and see if the CI passes nevertheless.

Reply:
Looks like all green. Therefore, removing.